client/package.json

```
@@ -63,6 +63,7 @@
     "chai": "4.3.4",
     "crypto-browserify": "3.12.0",
     "csrf": "3.1.0",
+    "css-tree": "^1.1.3",
```

```suggestion
    "css-tree": "1.1.3",
```
To avoid upsetting the bot.